site/sass/components/_body.phosphor.scss

```diff
@@ -0,0 +1,75 @@
+// Styles for the main sections
+/* stylelint-disable selector-no-qualifying-type */
+body[data-theme='phosphor'] {
+  //   animation: blur 30ms infinite, flick 30ms infinite, jump 30ms infinite;
+  background-color: var(--grey-50);
+
+  //   &::before {
+  //     animation: vhs 20s ease-in-out infinite, scanlines 0.2s ease-in-out alternate infinite;
+  //     background: linear-gradient(rgba(18, 16, 16, 0) 50%, rgba(0, 0, 0, 0.25) 50%), linear-gradient(90deg, rgba(255, 0, 0, 0.06), rgba(0, 255, 0, 0.02), rgba(0, 0, 255, 0.06));
+  //     background-size: 100% 5px, 3px 100%;
+  //     bottom: 0;
+  //     content: '';
+  //     height: 200%;
+  //     left: 0;
+  //     opacity: 0.8;
+  //     pointer-events: none;
+  //     position: absolute;
+  //     right: 0;
+  //     top: 0;
+  //     z-index: 1000;
+  //   }
+}
+
+[data-theme='phosphor'] .main {
+  //   margin-top: 4rem;
+}
+
+// @keyframes blur {
+//   0% {
+//     filter: blur(1px);
+//     opacity: 0.95;
+//   }
+
+//   50% {
+//     filter: blur(1.1px);
+//     opacity: 1;
+//   }
+
+//   100% {
+//     filter: blur(1px);
+//     opacity: 0.95;
+//   }
+// }
+// @keyframes flick {
+//   50% {
+//     left: 2px;
+//   }
+
+//   51% {
+//     left: 0;
+//   }
+// }
+// @keyframes jump {
+//   30% {
+//     top: 10px;
+//   }
+
+//   31% {
+//     top: 0;
+//   }
+// }
+// @keyframes vhs {
+//   0% {
+//     transform: translatey(0%);
+//   }
+
+//   100% {
+//     transform: translatey(-50%);
+//   }
+// }
+// @keyframes scanlines {
+//   to {
+//     opacity: 0.6;
+//   }
+// }
```

Comment:
Remove commented code

Reply:
updated/removed. there was some testing around adding CRT static styling, but abandoned for time. This was a remnant of that.